x-pack/plugins/ml/server/lib/alerts/jobs_health_service.ts

```diff
@@ -99,17 +131,36 @@ export function jobsHealthServiceProvider(
             const jobState =
               jobsStats.find((jobStats) => jobStats.job_id === jobId)?.state ?? 'failed';
             return {
-              ...datafeedStats,
+              datafeed_id: datafeedStats.datafeed_id,
+              datafeed_state: datafeedStats.state,
               job_id: jobId,
               job_state: jobState,
             };
           })
           .filter((datafeedStat) => {
             // Find opened jobs with not started datafeeds
-            return datafeedStat.job_state === 'opened' && datafeedStat.state !== 'started';
+            return datafeedStat.job_state === 'opened' && datafeedStat.datafeed_state !== 'started';
           });
       }
     },
+    /**
+     * Gets jobs that reached soft or hard model memory limits.
+     * @param jobIds
+     */
+    async getMmlReport(jobIds: string[]): Promise<MmlTestResponse[]> {
+      const jobsStats = await getJobStats(jobIds);
+
+      return jobsStats
+        .filter((j) => j.model_size_stats.memory_status !== 'ok')
+        .map(({ job_id: jobId, model_size_stats: modelSizeStats }) => {
+          return {
+            job_id: jobId,
+            memory_status: modelSizeStats.memory_status,
+            log_time: modelSizeStats.log_time,
+            failed_category_count: modelSizeStats.failed_category_count,
```

Comment:
As mentioned above, probably worth adding in at least `model_bytes_memory_limit`, `peak_model_bytes`, `model_bytes_exceeded`

Reply:
Added in 94a7d2803c917f044a871fd2644c44c92b50ccd7

Reply:
Can you add in `model_byte` too